workshops/forms.py

```diff
@@ -412,3 +413,18 @@ class Meta:
         model = Task
         fields = '__all__'
         widgets = {'person': HiddenInput}
+
+
+class HostForm(forms.ModelForm):
+    domain = forms.CharField(
+        max_length=Host._meta.get_field('domain').max_length,
+        validators=[
+            RegexValidator('[^\w\.-]+', inverse_match=True,
```

Comment:
Right, but why match inverse?  Why not match exact, so that the regexes 
can be the same?